language/tools/move-cli/src/package.rs

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{anyhow, bail, Result};
+use include_dir::{include_dir, Dir};
+use once_cell::sync::Lazy;
+use std::{
+    collections::HashSet,
+    fs::{self, File},
+    io::Write,
+    path::Path,
+};
+
+use move_lang::{
+    compiled_unit::CompiledUnit, extension_equals, find_filenames, move_compile, path_to_string,
+};
+use std::path::PathBuf;
+use stdlib::{COMPILED_EXTENSION, MOVE_EXTENSION};
+use vm::file_format::CompiledModule;
+
+/// Directory name for the package source files under package/<name>
+const PKG_SOURCE_DIR: &str = "source_files";
+/// Directory name for the package binary files under package/<name>
+const PKG_BINARY_DIR: &str = "compiled";
+
+/// Content for the stdlib directory
+const DIR_STDLIB: Dir = include_dir!("../../stdlib/modules");
+/// Content for the nursery directory
+const DIR_NURSERY: Dir = include_dir!("../../stdlib/nursery");
+
+/// The portion in the stdlib directory that are considered stdlib
+const STDLIB_MODULE_FILES: [&str; 9] = [
+    "CoreAddresses.move",
```

Comment:
I've remove CoreAddresses from this list as it's really defining the key addresses for the libra side of things.